src/docfx/build/link/LinkResolver.cs

```diff
@@ -107,6 +111,8 @@ internal class LinkResolver
             return (error, link, file);
         }
 
+        public IReadOnlyCollection<FilePath> GetAdditionalResources() => _additionalResources.ToHashSet();
```

Comment:
Eliminate the unneeded `ToHashSet` call
```suggestion
        public IEnumerable<FilePath> GetAdditionalResources() => _additionalResources;
```